client/functions.coffee

```diff
@@ -12,7 +12,7 @@
 		# secure_url
 		# relative_url
 
-	upload: (ops = {},callback) ->
+	upload: (ops = {},callback, progressCb) ->
```

Comment:
Mmm, not sure about this pattern, maybe we can have it inside of ops or build a separate reactive bound function to catch it. What do you think @gsabran ?